recipes/browndye/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = 'browndye' %}
+# Browndye uses date-based versioning
+# but tags use English formatting
+{% set version = "2022.3.7" %}
+{% set tag = "07-Mar-2022" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}.tar.gz
+  url: https://bitbucket.org/gahuber95/browndye2/get/{{ tag }}.tar.gz
+  sha256: 1a985a455ef691ce10b425aa7cec25acf0718787c7d22877a02db3d087e12f08
+  patches:
+    - makefile-env.patch
+    - ldflags.patch
+
+build:
+  number: {{ build }}
+  # win: not supported
+  # osx: requires apbs, not packaged for osx
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - binutils
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - ocaml
+  host:
+    - expat
+    - liblapack
+  run:
+    - apbs
+    - expat
+    - liblapack
```

Comment:
```suggestion
```

Both packages seem to have a run_exports ([lapack](https://github.com/conda-forge/lapack-feedstock/blob/main/recipe/meta.yaml#L150), [expat](https://github.com/conda-forge/expat-feedstock/blob/main/recipe/meta.yaml#L14)), so I don't think these lines are necessary. Can they be removed?

Reply:
Removed.